sparkler-app/src/main/scala/edu/usc/irds/sparkler/pipeline/Crawler.scala

```diff
@@ -172,6 +164,27 @@ class Crawler extends CliTool {
     LOG.info("Shutting down Spark CTX..")
     sc.stop()
   }
+
+  def score(fetchedRdd: RDD[CrawlData]): RDD[CrawlData] = {
+    val job = this.job
+
+    val scoredRdd = fetchedRdd.map(d => ScoreFunction(job, d))
+
+    val scoreUpdateRdd: RDD[SolrInputDocument] = scoredRdd.map(d => ScoreUpdateSolrTransformer(d))
+    val scoreUpdateFunc = new SolrStatusUpdate(job)
+    sc.runJob(scoreUpdateRdd, scoreUpdateFunc)
```

Comment:
@giuseppetotaro 
`StatusUpdateSolrTransformer` is swapped with `ScoreUpdateSolrTransformer`
Result: 
The crawled documents are not updated with `status=FETCHED` which uis

Haha, this one was really hard to catch!
Now we know what exactly is wrong, let's think of fixing it before merging.
1. We need to update `score` as well as `status` fields.
2. We need to get score from scorer plugins, status from fetcher plugin and then create one object `SolrInputDoc`. 

As you see, we get pieces of updates from plugins, we need to assemble one big update (ie. one `SolrInputDoc` with multiple `set` attributes) and pass it to solr.

Sorry, I am not trying to be hard on this PR, but without updating `status=FETCHED`, our crawler will be crawling the same pages over and over again. Hope you see it this way :-)